### ISSUE TITLE
TOC sidebar is out of date for v0.10 and main and missing for v0.11 release

### ISSUE THREAD

**Issue by stmcginnis** [vmware-tanzu/community-edition#3704]
## Description

When the versioned docs are generated, the sidebar for the docs are configured in `*-toc.yaml` files. Since we hadn't been working with versioned docs in the past, the v0.10 TOC contains some things that were not part of that release. The `main-toc.yaml` file contains different packages than what is currently in the repo.

Once `main-toc.yaml` is updated, that is likely going to match what the v0.11 release TOC should be, though that should be checked to make sure there aren't minor differences. But once reconciled, `main-toc.yaml` can probably be copied to be the TOC for the v0.11 release branch.

## Location

Documentation table of contents for each version.

## Additional Context

I think `main` is used for `edge`, but that should also be confirmed.

**Comment by kcoriordan** [comment]
@stmcginnis, once the back porting is complete, I think 2 above should be resolved with the step b below to 'publish' the docs on release date?

Steps to publish the docs
>a. In config.yaml:
>change docs_latest to v0.11
>add v0.11 to docs_versions

>b. Copy main-toc.yml and rename copy as v0.11-toc.yml
>Add new  row  in toc-mapping.yml: v0.11: v0.11-toc

For 1 above - I don't really know how to resolve this - I think it might be a separate issue?

**Comment by kcoriordan** [comment]
#3715 should close out part 2 above.
